### PR TITLE
Add QoS profiles for Arista SKUs

### PR DESCRIPTION
**Pull request by ysmanman**
#### Why I did it

QoS profile is missing for Arista SKUs like  Arista-7800R3AK-36DM2-C36.

#### How I did it

Add QoS profile for Arista-7800R3AK-36DM2-C36.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by ysmanman** [comment]
Add @vmittal-msft @arlakshm for awareness.

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "6441610000",
+            "type": "both",
+            "mode": "dynamic",
+            "xoff": "11354112"
```
Comment (vmittal-msft) [inline device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2:16]: Rest of the setting look ok. Can you please share calculation details for XOFF ? Also, are these profiles going to cover all Wolverine SKUs ?

**Review by vmittal-msft — APPROVED** [review]

File: device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2 (hunk 1 above)
Comment (rlhui) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2:16]: Hi @kenneth-arista , @ysmanman - reminder of this

Comment (vmittal-msft) [reply device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/buffers_defaults_t2.j2:16]: As per discussion, these settings are to unblock testing and not the final one. I will raise another PR to update these settings based on cable length we are planning to use.

**Comment by vmittal-msft** [comment]
@rlhui Can you please help merge this as i don't have permission ?

**Comment by vmittal-msft** [comment]
@lguohan @yxieca Can you please help merge this ? I don't have permission.

**Review by yxieca — APPROVED** [review]

**Pull request merged**